docs/macOS/macOS_supported_versions.md

```diff
@@ -1,11 +1,7 @@
 Compatible macOS versions for the EOS apk
 -------------------------------------------------------------------
 
+|Version|Standalone Build|Editor|
```

Comment:
The table formatting didn't go through.
Adding a line of |----------|---------|----------| 
should do the trick.

Reply:
Ah, good catch. You actually can just do single dashes - but that doesn't quite look as good in plane text format :\